lib/karafka/testing/rspec/helpers.rb

```diff
@@ -44,6 +44,7 @@ def included(base)
 
               _karafka_consumer_messages.clear
               _karafka_producer_client.reset
+              @_karafka_consumer_mapping = {}
```

Comment:
can we use plural? `_karafka_consumers_mappings` to indicate it may hold more than one?